docs/integrations/sources/shopify.md

```diff
@@ -125,40 +60,44 @@ Add the following scopes to your custom app to ensure Airbyte can sync all avail
 * `read_order_edits`
 * `read_orders`
 
-### Output Streams Schemas
+## Supported sync modes
+
+The Shopify source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+This source can sync data for the [Shopify REST API](https://shopify.dev/api/admin-rest) and the [Shopify GraphQl API](https://shopify.dev/api/admin-graphql).
+
+## Troubleshooting tips
+
+Check out common troubleshooting issues for the Shopify source connector on our Discourse [here](https://discuss.airbyte.io/tags/c/connector/11/source-shopify).
+
+## Supported Streams
 
 This Source is capable of syncing the following core Streams:
 
-* [Articles](https://shopify.dev/api/admin-rest/2022-01/resources/article)
```

Comment:
how come you removed articles and blogs?

Reply:
Thanks for catching this. They were in one list but not the other, and I must have taken the older list. Added back.